indico/modules/rb_new/blueprint.py

```diff
@@ -43,3 +43,4 @@
 _bp.add_url_rule('/api/booking/create', 'create_booking', backend.RHCreateBooking,
                  methods=('POST',))
 _bp.add_url_rule('/api/suggestions', 'suggestions', backend.RHRoomSuggestions)
+_bp.add_url_rule('/rooms-sprite__v<version>.jpg', 'sprite', backend.RHRoomsSprite)
```

Comment:
I'd avoid the `__vXXX` since we use that for special handling in nginx rules.
Instead I'd go for a simple `rooms-sprite-<version>.jpg`.